lighthouse-core/lib/i18n/i18n.js

```diff
@@ -117,6 +119,29 @@ const formats = {
   },
 };
 
+
+/**
+ * @param {string} localePath
+ * @param {string} rootPath
+ */
+function mergeLocales(localePath, rootPath) {
+  const fullLocalePath = `${rootPath}/${localePath}`;
+  fs.readdirSync(fullLocalePath).forEach((tempPath) => {
+    const locale = tempPath.replace(/\.json$/, '');
+    const relativePrefix = path.relative(LH_ROOT, rootPath).replace(/\\/g, '/');
+    if (LOCALES[locale]) {
+      const localeToMerge = require(`${fullLocalePath}/${tempPath}`);
```

Comment:
can you separate this inner bit into its own function?   

`appendStringsToLocale` ?